src/test/java/core/basesyntax/service/impl/ParseServiceImplTest.java

```diff
@@ -0,0 +1,43 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.ParseService;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Test;
+
+public class ParseServiceImplTest {
+    private ParseService parseService = new ParseServiceImpl();
+    private List<String> dataFromFile = new ArrayList<>();
+    private List<FruitTransaction> expectedData = new ArrayList<>();

```

Comment:
Why do you need declaration these variables here?  In this case, better creates private methods that return some test data.